extra/linux/org.alacritty.Alacritty.appdata.xml

```diff
@@ -1,34 +1,45 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Copyright 2016-2022 Joe Wilm, The Alacritty Project Contributors -->
+<!-- SPDX-License-Identifier: MIT -->
 <component type="desktop-application">
   <id>org.alacritty.Alacritty</id>
-  <!-- Translators: The application name -->
+
+  <developer_name>Christian Duerr</developer_name>
+
   <name>Alacritty</name>
-  <project_license>APACHE-2.0</project_license>
-  <metadata_license>APACHE-2.0</metadata_license>
-  <!-- Translators: The application's summary / tagline -->
-  <summary>A fast, cross-platform, OpenGL terminal emulator</summary>
+
+  <project_license>Apache-2.0</project_license>
+  <metadata_license>MIT</metadata_license>
```

Comment:
Why go with MIT here? Just because it's most compatible with Apache-2.0? I personally don't care about anything in here so I do think that cc0 would be fine too.

Reply:
yeah, and we have a license file around.

Reply:
A license file isn't really necessary.